webrender/src/device/gfx.rs

```diff
@@ -1636,37 +1643,37 @@ impl<B: hal::Backend> DescriptorPools<B> {
         let cache_clip_range = vec![
             hal::pso::DescriptorRangeDesc {
                 ty: hal::pso::DescriptorType::SampledImage,
-                count: 400,
+                count: descriptor_count * 10,
             },
             DescriptorRangeDesc {
                 ty: hal::pso::DescriptorType::Sampler,
-                count: 400,
+                count: descriptor_count * 10,
```

Comment:
do we have to have this limit as constant? Wouldn't it need to grow with the content complexity?